.github/pull_request_template.md

```diff
@@ -1,12 +1,14 @@
 ## Description
 
-Include a summary of the changes and a link to the related issue. List any dependencies that are required for this change.
+Summary of changes:
 
 [//]: # 'remove if not applicable'
 Depends on hmis-warehouse PR:
 
+[//]: # 'remove if not applicable'
+How to test:
+
 ## Type of change
-[//]: # 'remove options that are not relevant'
```

Comment:
I do usually do this, so it doesn't look like the PR has unfinished tasks. Thats probably why this note was added. I think its oK to keep it
<img width="345" alt="Screenshot 2025-03-27 at 4 47 40 PM" src="https://github.com/user-attachments/assets/4e55576c-f2dc-48bb-96d0-d215036a0c4b" />